Makefile

```diff
@@ -2,7 +2,7 @@ VERSION := $(shell git describe --tags)
 DIST_DIRS := find * -type d -exec
 
 build:
-	go build -o glide -ldflags "-X main.version=${VERSION}" glide.go
+	go build -o glide -ldflags "-X main.version=$(VERSION)" glide.go
```

Comment:
> As the one who runs the builds, the current implementation generates the correct version. Are you seeing something different?

This is more a matter of style and portability to non-POSIX shell than an error.